crates/ide_assists/src/handlers/add_missing_impl_members.rs

```diff
@@ -140,6 +150,18 @@ fn add_missing_impl_members_inner(
     })
 }
 
+fn try_gen_trait_body(
+    ctx: &AssistContext,
+    func: &ast::Fn,
+    trait_: &hir::Trait,
+    impl_def: &ast::Impl,
+) -> Option<()> {
+    let trait_path = make::path_from_text(&trait_.name(ctx.db()).to_string());
```

Comment:
Let's not use `path_from_text` here, this function shouldn't be public in the first place, instead use `make::ext::ident_path`.